src/views/Search/Records.vue

```diff
@@ -1,13 +1,18 @@
 <template>
-    <div class="outputTable">
+    <div class="outputGrid" v-if="client">
         <h1>{{currentPath}}</h1>
-        <output-table>
-        </output-table>
+        <form>
+            <input type="text" v-model="searchString">
+            <button type="button" @click="getData()">Search</button>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['@click' should be on a new line.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4893952)